src/ducks/billsMatching/Linker/billsToOperation/operationsFilters.spec.js

```diff
@@ -177,14 +172,14 @@ describe('operations filters', () => {
         label: 'Visite chez le médecin',
         _id: 'o1',
         date: new Date(2017, 11, 13),
-        automaticCategoryId: '400610'
+        manualCategoryId: '400610'
```

Comment:
Question: why do we change from automatic to manual.

Reply:
See https://github.com/cozy/cozy-banks/pull/1498#discussion_r317960624